tests/documentation_tests/scripts/prebuilt-func.sh

```diff
@@ -194,3 +198,72 @@ pods_istio_version_match() {
 
     return 0
 }
+
+# Check that the provided istiorevision is in use
+istio_active_revision_match() {
+    expected_revision="$1"
+
+    active_revision=$(kubectl get istio -o jsonpath='{.items[0].status.activeRevisionName}')
+
+    if [ "$active_revision" != "$expected_revision" ]; then
+        echo "Expected active revision $expected_revision, got $active_revision"
+        return 1
+    fi
+
+    echo "Active revision is $active_revision"
+}
+
+# Check that the number of istiorevisions ready match the expected number
+istio_revisions_ready_count() {
+    expected_count="$1"
+
+    revisions=$(kubectl get istio -o jsonpath='{.items[0].status.revisions.ready}')
+
+    if [ "$revisions" -ne "$expected_count" ]; then
+        echo "Expected $expected_count istiorevisions, got $revisions"
+        return 1
+    fi
+
+    echo "Found $revisions istiorevisions"
+}
+
+# Check the number of istiod pods running in the istio-system namespace
+istiod_pods_count() {
+    expected_count="$1"
+
+    pods=$(kubectl get pods -n istio-system -l app=istiod -o jsonpath='{.items[*].status.phase}')
```

Comment:
Following might be used too: `kubectl get pods -n istio-system -l app=istiod --output json | jq -j '.items | length'`